slacker/logger.py

```diff
@@ -42,8 +43,8 @@ def __stream_handler(self):
     sh.setLevel(self.log_level)
     return sh
 
-  def __configure_formatter(self, logger):
-    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+  def __configure_formatter(self, logger, fmt):
```

Comment:
Would it make sense to add the default format to be `'%(asctime)s - %(name)s - %(levelname)s - %(message)s'` here. That way if we create another log handler it is used by default, I personally like the log format when not being used for stdout.

Reply:
I will do that. Good point.